builds.sh

```diff
@@ -19,13 +19,26 @@ shell="$(jq -r '
 ' "$json")"
 eval "$shell"
 
+_resolveRemoteArch() {
+	local img="$1"; shift
+
+	local arches
+	if ! arches="$(bashbrew remote arches --json "$img" 2>/dev/null)"; then # TODO somehow differentiate errors like 404 / 403, "insufficient_scope" from other bashbrew errors here so we can stop eating stderr
+		return
+	fi
+
```

Comment:
Perhaps we should be caching here rather than repeating the code twice in the two places where this function is called.

Reply:
That was what I tried at first but there's a lot of subshells going in on and the cache gets lost if we put it here. That's why it ended where it is now.

Reply:
Ah yes, caching a returned value is tricky in shell.